service/index.js

```diff
@@ -25,12 +25,12 @@ const getProfile = (userId) => {
     });
 };
 
-const putEditProfile = (userId, first_name, last_name, age, snack, animal_type) => {
+const putEditProfile = (email, first_name, last_name, age, snack) => {
   return pool
     .connect()
     .then((client) => {
-      const query = 'UPDATE users SET last_name = $3, age = $4 WHERE first_name = $2'; // to update userId params
-      const values = [userId, first_name, last_name, age, snack];
+      const query = 'UPDATE users SET first_name = $2, last_name = $3, age = $4, snack = $5, animal_type = $6 WHERE email = $1'; // to update userId params
```

Comment:
This is a better implementation than checking by first_name for sure. Love seeing animal_type too, will add it to my fake data  👌

Reply:
Awesomes! :)